rules/TaskManyArgs.py

```diff
@@ -0,0 +1,18 @@
+from ansiblelint import AnsibleLintRule
+
+class TaskManyArgs(AnsibleLintRule):
+    id = 'E303'
+    shortdesc = 'Use ":" YAML format when arguments are over 3'
+    description = ''
+    tags = ['task']
+
+    def match(self, file, text):
+        count = 0
```

Comment:
Note sure if it's more readable, but this can be written as:

```python
count = len([part for part in text.split(" ") if "=" in part])
return count > 3
```